devicetypes/smartthings/smartsense-temp-humidity-sensor.src/smartsense-temp-humidity-sensor.groovy

```diff
@@ -138,10 +139,20 @@ def ping() {
 
 def refresh() {
 	log.debug "refresh temperature, humidity, and battery"
-	return zigbee.readAttribute(0xFC45, 0x0000, ["mfgCode": 0x104E]) +   // New firmware
-			zigbee.readAttribute(0xFC45, 0x0000, ["mfgCode": 0xC2DF]) +   // Original firmware
-			zigbee.readAttribute(zigbee.TEMPERATURE_MEASUREMENT_CLUSTER, 0x0000) +
-			zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0020)
+
+	def manufacturer = device.getDataValue("manufacturer")
+	log.debug "${manufacturer}"
```

Comment:
If you think this log message is necessary can you add it to the log at the top of the function (ex: `refresh temperature, humidity and battery for manufacturer Heiman`. Otherwise you can just remove this message altogether.